polaris-react/src/components/Backdrop/Backdrop.scss

```
@@ -9,8 +9,9 @@ $entry-iterations: 1;
   left: 0;
   display: block;
   background-color: var(--p-backdrop);
-  animation: fade-in var(--p-duration-200) $entry-iterations forwards;
-  opacity: 1;
+  animation: var(--p-animation-name-fade-in) var(--p-duration-200)
+    $entry-iterations forwards;
```

```suggestion
    1 forwards;
```

Could we remove `$entry-iterations` it doesn't seem to add a lot of value?

---

💯 Good call!